File: openGl/java/GlHandlePeer.java
//
// Copyright (c) 2011, chunquedong
// Licensed under the Academic Free License version 3.0
//
// History:
//   2011-5-31  Jed Young  Creation
//

package fan.fanvasOpenGl;


class GlHandlePeer
{
  private int value;

  public static GlHandlePeer make(GlHandle self){ return new GlHandlePeer(); }

  public Object val(GlHandle self){ return value; }

  public int getValue(){ return value; }
  public void setValue(int v){ value = v; }
}

File: openGl/js/GlHandlePeer.js
//
// Copyright (c) 2011, chunquedong
// Licensed under the Academic Free License version 3.0
//
// History:
//   2011-5-29  Jed Young  Creation
//


fan.vaseOpenGl.GlHandlePeer = fan.sys.Obj.$extend(fan.sys.Obj);
fan.vaseOpenGl.GlHandlePeer.prototype.$ctor = function(self) {}
fan.vaseOpenGl.GlHandlePeer.prototype.value = null;
fan.vaseOpenGl.GlHandlePeer.prototype.val = function(self){ return this.value; }

fan.vaseOpenGl.GlHandlePeer.prototype.getValue = function(){ return this.value; }
fan.vaseOpenGl.GlHandlePeer.prototype.setValue = function(v){ this.value = v; }

